swaywm/wlroots
# Add map/unmap to xdg-shell

- [x] Add unmap support to zxdg-shell v6
- [x] Read client requested state in `new_surface` in zxdg-shell-v6
- [x] Add map/unmap support to xdg-shell stable

Test plan:
* Make sure xdg-shell still works
* Try `mpv --fs` or https://gist.github.com/emersion/34ed72adecd78bdfac9208a97ffb46a3
* Try to reproduce crashes described in issues below

Fixes #688
Fixes #674
Fixes #679
Fixes #671
Fixes #650
Closes #681
Closes #708
